ta4j-core/src/test/java/org/ta4j/core/analysis/criteria/AverageProfitableTradesCriterionTest.java

```diff
@@ -55,16 +55,16 @@ public void calculateWithOneTrade() {
         Trade trade = new Trade(Order.buyAt(0, series), Order.sellAt(1, series));
             
         AverageProfitableTradesCriterion average = new AverageProfitableTradesCriterion();
-        assertEquals(0d, average.calculate(series, trade), TestUtils.GENERAL_OFFSET);
+        assertNumEquals(numOf(0), average.calculate(series, trade));
         
         trade = new Trade(Order.buyAt(1, series), Order.sellAt(2, series));
-        assertEquals(1d, average.calculate(series, trade), TestUtils.GENERAL_OFFSET);
+        assertNumEquals(1, average.calculate(series, trade));
     }
 
     @Test
     public void betterThan() {
         AnalysisCriterion criterion = new AverageProfitableTradesCriterion();
-        assertTrue(criterion.betterThan(12, 8));
-        assertFalse(criterion.betterThan(8, 12));
+        assertTrue(criterion.betterThan(numOf(12), numOf(8)));
+        assertFalse(criterion.betterThan(numOf(8), numOf(12)));
     }
 }
```

Comment:
AverageProfitableTrades Test looks good.  Am wondering if the `double` calculations like `2d/3` should be replaced with `Num` calculations..